tasks/main.yml

```diff
@@ -16,12 +16,14 @@
 - name: Install and configure required providers
   vars:
     __certificate_providers: |
-      {{
-        certificate_requests |
-        map(attribute='provider', default=__certificate_provider_default) |
-        unique |
-        list
-      }}
+      {% set ns = namespace(unique_providers=[]) %}
```

Comment:
I never used used `namespace`, why is it required?

Reply:
https://jinja.palletsprojects.com/en/2.10.x/templates/#assignments - see the entire "Scoping Behavior" section